synapse/third_party_rules/access_rules.py

```diff
@@ -33,6 +33,11 @@
     ACCESS_RULE_UNRESTRICTED,
 )
 
+# Rules to which we need to apply the power levels restrictions.
+RULES_WITH_RESTRICTED_POWER_LEVELS = (
+    ACCESS_RULE_UNRESTRICTED,
```

Comment:
I think we could do with a few words about why PL changes are restricted in unrestricted rooms. I mean, it makes sense when you think about it, but at first glance it seems surprising.

(I'd actually be tempted to get rid of `RULES_WITH_RESTRICTED_POWER_LEVELS` and just have `if rule == ACCESS_RULE_UNRESTRICTED` below, but ymmv)

Reply:
> (I'd actually be tempted to get rid of RULES_WITH_RESTRICTED_POWER_LEVELS and just have if rule == ACCESS_RULE_UNRESTRICTED below, but ymmv)

Me too, but we might want to add new rules that may or may not implement this restriction in the future.